pkg/minishift/network/utils.go

```diff
@@ -37,6 +37,11 @@ func GetIP(driver drivers.Driver) (string, error) {
 }
 
 func DetermineHostIP(driver drivers.Driver) (string, error) {
+	// In case of generic driver we don't need to get the bridge network
+	// since it's by default ping able from host IP.
+	if driver.DriverName() == "generic" {
+		return "", nil
```

Comment:
maybe return an Error here? they aren't in the same network ... This is a tricky one.